runtime/README.md

```diff
@@ -711,6 +711,84 @@ Command Code: `0x434B_4558` ("CKEX")
 | fips\_status        | u32       | Indicates if the command is FIPS approved or an error.
 | certify\_key\_resp  | u8[2176]  | Certify Key Response.
 
+### SET\_AUTH\_MANIFEST
+
+Command Code: `0x4154_4D4E` ("ATMN")
+
+*Table: `SET\_AUTH\_MANIFEST` input arguments*
+
+| **Name**                      | **Type**     | **Description**                                                             |
```

Comment:
This table is missing the header line so I think it renders incorrectly

Reply:
Updated.